webapp/crashstats/settings/bundles.py

```diff
@@ -34,12 +34,15 @@
     "@fortawesome/fontawesome-free": ["css/all.min.css", "webfonts/*"],
     "tablesorter": ["dist/css/theme.default.min.css", "dist/js/jquery.tablesorter.js"],
     "d3": ["dist/*"],
-    "jssha": ["dist/*.js"],
+    "jssha": ["dist/*.js", "dist/*.map"],
```

Comment:
The `.js` file has an indicator to the `.map` file and something uses that now (I have no idea what), so we need to copy over the `.map` file now.